### ISSUE TITLE
Icon request: mobile-phone

### ISSUE THREAD

**Issue by equinoxe-rehm** [ReduxFramework/elusive-icons#14]
Usecases: Contactform, Contact-Information in the Footer
![icon-mobilephone](https://cloud.githubusercontent.com/assets/13252810/8592518/411c9fa2-2630-11e5-8740-08c257978634.png)

Would be really great, I miss thet really often...

**Comment by marcvangend** [comment]
Yes, I support this request. In some cases you need to differentiate between a normal phone number and a mobile phone number.
